storybook/directorieswatcher.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <QObject>
+
+class QFileSystemWatcher;
+
+class DirectoriesWatcher : public QObject
+{
+    Q_OBJECT
+public:
+    explicit DirectoriesWatcher(QObject *parent = nullptr);
+    void addPaths(const QStringList &paths);
+
+signals:
+    void changed();
+
+private:
+    QFileSystemWatcher* fsWatcher;
```

Comment:
Why this is a pointer? There is no need to create it using new.
```suggestion
    QFileSystemWatcher fsWatcher;
```

Reply:
That's fair enough, although I am not a big fan of this solution (raw pointers in general, even in qtish environment). I prefer readability and maintainability over compilation times. Here, you don't know straight away who the owner of `QFileSystemWatcher` is.

Reply:
yeah, there are some pros/cons...

Reply:
Yup, those raw pointers... :) I'd prefer either a stack var, or forward decl too